tests/selenium.go

```diff
@@ -0,0 +1,598 @@
+/*
+ * Copyright (C) 2018 Skydive-Project Authors.
```

Comment:
nit: you should probably use the same than `tests.go`

Reply:
fixed